images/linux/toolsets/toolset-1804.json

```diff
@@ -44,7 +44,8 @@
             "versions": [
                 "1.16.*",
                 "1.17.*",
-                "1.18.*"
+                "1.18.*",
```

Comment:
@erik-bershel @ddobranic I suggest not adding Go 1.19 to Ubuntu 18 as the image will start its' deprecation process on Monday and we can persuade users to migrate to the newer Ubuntu version by not adding the new tool versions